### PR TITLE
Misc2

### PR DESCRIPTION
**Pull request by kgghfc**
forgot description

### CODE REVIEW

File: app/src/main/java/com/cmput301/cmput301w127t06/moodbook/AddEventActivity.java [hunk 1]
```diff
@@ -217,10 +224,31 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
...
             event.setPhoto(imageBitmap);
+
+            ImageView thumb = (ImageView) findViewById(R.id.image);
+            thumb.setImageBitmap(getResizedBitmap(imageBitmap, 350, 350));
+
         }
     }
+
+    // http://stackoverflow.com/a/10703256
+    public Bitmap getResizedBitmap(Bitmap bm, int newWidth, int newHeight) {
```
Comment (Everlag) [inline app/src/main/java/com/cmput301/cmput301w127t06/moodbook/AddEventActivity.java]: Nice

File: app/src/main/java/com/cmput301/cmput301w127t06/moodbook/Comment.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    public String getPostedBy() {
+        return postedBy;
+    }
+
+    public void setPostedBy(String postedBy) {
+        this.postedBy = postedBy;
+    }
+
+    @Override
+    public String toString() {
```
Comment (Everlag) [inline app/src/main/java/com/cmput301/cmput301w127t06/moodbook/Comment.java]: Line's a bit long

File: app/src/main/java/com/cmput301/cmput301w127t06/moodbook/CommentActivity.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        username = intent.getStringExtra("USERNAME");
+        event = intent.getParcelableExtra("MOODEVENT");
+        commentsArrayList = event.getComments();
+
+        comments = (ListView) findViewById(R.id.comments);
+
+        final EditText field = (EditText) findViewById(R.id.message);
+        sendButton = (Button) findViewById(R.id.send);
+
+        final ArrayAdapter<Comment> commentArrayAdapter = new ArrayAdapter<Comment>(this, R.layout.item_comment, commentsArrayList);
```
Comment (Everlag) [inline app/src/main/java/com/cmput301/cmput301w127t06/moodbook/CommentActivity.java]: Break the line please

File: app/src/main/java/com/cmput301/cmput301w127t06/moodbook/MoodData.java [hunk 4]
```diff
@@ -21,14 +21,22 @@
...
-        put("disgusted", Color.GREEN);
-        put("scared", Color.DKGRAY);
-        put("happy", Color.MAGENTA);
-        put("sad", Color.YELLOW);
-        put("ashamed", Color.BLUE);
-        put("surprised", Color.CYAN);
+        // pink
+        put("angry", 0xffffcece); //
+        // orange
+        put("confused", 0xffffe5ce);
```
Comment (ZackerySpytz) [inline app/src/main/java/com/cmput301/cmput301w127t06/moodbook/MoodData.java]: These colors are nicer.

**Pull request merged**